<launch>


  <include file="$(find turtlebot_stage)/launch/turtlebot_in_stage.launch"/>

  <node pkg="exercise1" type="stopper.py" name="stopper" output="screen">
    <param name="stop_dist" value="0.5"></param>
    <param name="max_speed" value="0.1"></param>
  </node>


</launch>